### PR TITLE
Port cart recovery

### PR DESCRIPTION
**Pull request by aasalazar23**
Checkout Recovery Screen
Screen Shot 2021-06-01 at 14 52 55 PM

With error
Screen Shot 2021-06-01 at 14 34 37 PM

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -92,4 +92,10 @@ exports.onCreatePage = ({ page, actions }) => {
     page.matchPath = "/sign_up/*";
     actions.createPage(page);
   }
+
+  // cart recovery
+  if (page.path.match(/^\/checkout/)) {
+    page.matchPath = "/checkout/*";
+    actions.createPage(page);
+  }
```
Comment (danielpowell4) [inline gatsby-node.js:100]: All good here!

Just want to confirm that the thank you page and any required GTag events are also good to go

File: src/components/CollectRetrievedPayment.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ import ThreeDotLoader from "./shared/three-dot-loader";
 import { loadStripe } from "@stripe/stripe-js";
 
 import "./CollectRetrievedPayment.css";
```
Comment (danielpowell4) [inline src/components/CollectRetrievedPayment.js:15]: if these styles are effectively dead now is a good time for 🗑️

**Comment by danielpowell4** [comment]
Don't know where to put this but seems like it would be a good exercise to see what these tests are "actually" giving us and making sure they include all the sorts of thing we'd want them to

**Comment by danielpowell4** [comment]
Per your CSS + photos is it easy to change the background color of the `🚩` in the `<details>` of the error message?

Maybe something like

```css
details summary::marker {
  color: currentColor;
}
```

**Comment by danielpowell4** [comment]
is this waiting on anything?

**Comment by aasalazar23** [comment]
> is this waiting on anything?

Wanted to test it with you per our conversation yesterday about not knowing where the stripe endpoint was going.

**Pull request merged**